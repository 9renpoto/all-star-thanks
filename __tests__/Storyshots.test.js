/* @flow */
const initStoryshots = require('@storybook/addon-storyshots').default
const faker = require('faker')
faker.seed(123)
initStoryshots({
  storyKindRegex: /^((?!.*?DontTest).)*$/
})
