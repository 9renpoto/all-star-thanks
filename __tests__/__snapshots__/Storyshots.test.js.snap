// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Content App 1`] = `
<ul>
  <li>
    <span
      className="num"
    >
      10
    </span>
    <span
      className="name"
    >
      Swedish Krona
    </span>
    <span
      className="point"
    >
      286
      <small>
        P
      </small>
    </span>
    <span
      className="time"
    >
      7/7/2018
    </span>
  </li>
  <li>
    <span
      className="num"
    >
      9
    </span>
    <span
      className="name"
    >
      Quality-focused
    </span>
    <span
      className="point"
    >
      551
      <small>
        P
      </small>
    </span>
    <span
      className="time"
    >
      3/23/2018
    </span>
  </li>
  <li>
    <span
      className="num"
    >
      8
    </span>
    <span
      className="name"
    >
      sensor
    </span>
    <span
      className="point"
    >
      423
      <small>
        P
      </small>
    </span>
    <span
      className="time"
    >
      3/1/2018
    </span>
  </li>
  <li>
    <span
      className="num"
    >
      7
    </span>
    <span
      className="name"
    >
      Administrator
    </span>
    <span
      className="point"
    >
      684
      <small>
        P
      </small>
    </span>
    <span
      className="time"
    >
      5/13/2018
    </span>
  </li>
  <li>
    <span
      className="num"
    >
      6
    </span>
    <span
      className="name"
    >
      Crossing
    </span>
    <span
      className="point"
    >
      392
      <small>
        P
      </small>
    </span>
    <span
      className="time"
    >
      7/18/2018
    </span>
  </li>
  <li>
    <span
      className="num"
    >
      5
    </span>
    <span
      className="name"
    >
      extensible
    </span>
    <span
      className="point"
    >
      729
      <small>
        P
      </small>
    </span>
    <span
      className="time"
    >
      8/15/2018
    </span>
  </li>
  <li>
    <span
      className="num"
    >
      4
    </span>
    <span
      className="name"
    >
      communities
    </span>
    <span
      className="point"
    >
      59
      <small>
        P
      </small>
    </span>
    <span
      className="time"
    >
      4/1/2018
    </span>
  </li>
  <li>
    <span
      className="num"
    >
      3
    </span>
    <span
      className="name"
    >
      engage
    </span>
    <span
      className="point"
    >
      737
      <small>
        P
      </small>
    </span>
    <span
      className="time"
    >
      4/24/2018
    </span>
  </li>
  <li>
    <span
      className="num"
    >
      2
    </span>
    <span
      className="name"
    >
      Table
    </span>
    <span
      className="point"
    >
      175
      <small>
        P
      </small>
    </span>
    <span
      className="time"
    >
      11/10/2018
    </span>
  </li>
  <li>
    <span
      className="num"
    >
      1
    </span>
    <span
      className="name"
    >
      Poland
    </span>
    <span
      className="point"
    >
      531
      <small>
        P
      </small>
    </span>
    <span
      className="time"
    >
      7/8/2018
    </span>
  </li>
</ul>
`;
